csharp-architectural-patterns/ModularMonolithPatternInCSharp/Common/RabbitMq/IRabbitMqConnectionManager.cs

```diff
@@ -0,0 +1,13 @@
+﻿using RabbitMQ.Client;
+
+namespace Common.RabbitMq
+{
+    public interface IRabbitMqConnectionManager
+    {
+        IModel? Channel { get; set; }
```

Comment:
I suppose we don't want to have the setter `public`. Should be `private`. Same for the `Connection`.